lndr-backend/data/lndr-server.config.j2

```diff
@@ -59,4 +59,9 @@ notifications {
   api-key = "{{NOTIFICATIONS_API_KEY}}"
 }
 
+sumsub {
+  api-url = "{{SUMSUB_API_URL}}"
+  api-key = "{{SUMSUB_API_KEY}}"
+}
```

Comment:
**Note:** Before merging this PR, we'll need to add this configuration to the Ansible role and wire it up through the role's defaults and playbook's *group_vars*.

Reply:
This was completed in https://github.com/blockmason/infrastructure-playbook/pull/22.